drivers/storage/portworx/component/security.go

```diff
@@ -89,11 +89,20 @@ func (c *security) Initialize(
 
 // IsEnabled checks if the components needs to be enabled based on the StorageCluster
 func (c *security) IsEnabled(cluster *corev1.StorageCluster) bool {
-	return pxutil.SecurityEnabled(cluster)
+	return pxutil.AuthEnabled(&cluster.Spec) || pxutil.IsTLSEnabledOnCluster(&cluster.Spec)
 }
 
 // Reconcile reconciles the component to match the current state of the StorageCluster
 func (c *security) Reconcile(cluster *corev1.StorageCluster) error {
+	if pxutil.AuthEnabled(&cluster.Spec) {
+		logrus.Infof("Auth is enabled ... creating secrets")
```

Comment:
```suggestion
```

Reply:
This log will cause a lot of spam in the operator logs, I think we can remove this. This log happens everytime the storagecluster obj is edited or checked.